MapleServer2/Managers/TrophyManager.cs

```diff
@@ -85,66 +85,55 @@ public static void OnTrigger(Player player, string trigger)
         IEnumerable<TrophyMetadata> triggerTrophies = GetRelevantTrophies(TrophyTypes.Trigger);
 
         IEnumerable<TrophyMetadata> matchingTrophies = triggerTrophies
-            .Where(t => IsMatchingCondition(t.Grades.First().ConditionCodes, trigger));
+            .Where(t => IsMatching(t.Grades.First().ConditionCodes, trigger));
 
         UpdateMatchingTrophies(player, matchingTrophies, 1);
     }
 
-    private static IEnumerable<TrophyMetadata> GetRelevantTrophies(string category) =>
-        TrophyMetadataStorage.GetTrophiesByType(category);
+    #region Helper Methods
 
-    private static bool IsMatchingCondition(IEnumerable<string> trophyConditions, string condition)
+    private static IEnumerable<TrophyMetadata> GetRelevantTrophies(string category) => TrophyMetadataStorage.GetTrophiesByType(category);
+
+    /// <summary>
+    /// Checks if the given condition code or target match the given string value.
+    /// </summary>
+    /// <param name="trophyCondition">Trophy code or target</param>
+    /// <param name="value">Value to try match</param>
+    /// <returns>True or false</returns>
```

Comment:
Big ❤️ for including XML documentation, anything that helps intellisense is a win in my book.
I think it's definitely worth having a talk about what level of detail we want from these. For example, IMHO "Returns true or false" is obvious enough by nature of the method returning a boolean, and so I wouldn't have added the `<returns>` tag myself. Thank you for taking charge and starting positive change 😉